### ISSUE TITLE
utility-types is missing in peer dependencies

### ISSUE THREAD

**Issue by valentinpalkovic** [hinok/react-router-last-location#19]
Hello :)

In the current 2.0.0-beta.1 you are using `utility-types`, which is only used as `dev-dependency`. Please consider to move it in the `peer-dependency` or `dependency` config in the package json.

Following error occurs otherwise:
```
TypeScript error in /app/frontend/node_modules/react-router-last-location/dist/withLastLocation.d.ts(7,186):
Cannot find module 'utility-types'.  TS2307
5 |     lastLocation: ReturnType<typeof getLastLocation>;
6 | }
7 | declare const withLastLocation: <WrappedProps extends WithLastLocationProps>(WrappedComponent: React.ComponentType<WrappedProps>) => React.ComponentClass<Pick<Pick<WrappedProps, import("utility-types").SetDifference<keyof WrappedProps, "history" | "location" | "match" | "staticContext" | "lastLocation">> & RouteComponentProps<{}, import("react-router").StaticContext, any>, Exclude<import("utility-types").SetDifference<keyof WrappedProps, "history" | "location" | "match" | "staticContext" | "lastLocation">, "history" | "location" | "match" | "staticContext">>, any>;
```

**Comment by hinok** [comment]
Hey @valentinpalkovic ,
Many thanks for reporting the issue.
I published `2.0.0-beta.2` which should solve the issue. You don't have to install `utility-types`, just upgrade `react-router-last-location`.

**Comment by hinok** [comment]
I'm closing issue because this package doesn't depend anymore on `utility-types`.
